src/main/java/com/robosh/ejournal/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package com.robosh.ejournal.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Data
+@NoArgsConstructor
+@Builder
+@Entity
+public class User {
+    @Id
+    @GeneratedValue
+    protected Long id;
+    protected String firstName;
+
+    protected String secondName;
+
+    protected String lastName;
+
+    protected String email;
+
+    protected String password;
+
+    protected String telephone;
+
+    @ManyToOne
+    private School school;
```

Comment:
why this field is private? maybe we need to make it protected?

Reply:
Probably